x/gov/migrations/v4/store.go

```diff
@@ -62,6 +68,55 @@ func migrateProposalVotingPeriod(ctx sdk.Context, storeKey storetypes.StoreKey,
 	return nil
 }
 
+// AddProposerAddressToProposal will add proposer to proposal
+// and set to the store. This function is optional, and only needed
+// if you wish that migrated proposals be cancellable.
+func AddProposerAddressToProposal(ctx sdk.Context, storeKey storetypes.StoreKey, cdc codec.BinaryCodec, proposals map[uint64]string) error {
```

Comment:
Unfortunately this won't go into v0.47, which has gov@v4. Could you create a v5 folder, and add this to a v4->v5 migration?